core/src/main/java/org/apache/accumulo/core/iterators/system/LocalityGroupIterator.java

```diff
@@ -16,23 +16,28 @@
  */
 package org.apache.accumulo.core.iterators.system;
 
+import com.google.common.collect.ImmutableSet;
```

Comment:
Can you please revert the reordering of imports?

If the imports are actually in the wrong order, please open up another issue and lets squash them all at once.